site/en/docs/extensions/mv3/manifest/content_scripts/index.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Manifest - Content Scripts'
```

Comment:
```suggestion
title: 'Manifest - content scripts'
```

Reply:
Here's my reasoning. Titles are sentence case. There is no rule for capitalization after a dash or hyphen, or at leaste none that I can find. However, there is a rule for capitalization after a colon. (Chicago Manual of Style, 17th ed., rule 6.63.) Hyphens and colons have overlapping and analogous functions, so 6.63 applies.